Sources/SwiftSyntaxBuilder/ConvenienceInitializers.swift

```diff
@@ -13,6 +13,26 @@
 @_spi(RawSyntax) import SwiftParser
 @_spi(RawSyntax) import SwiftSyntax
 
+// MARK: - ArrayElementList
+
+extension ArrayElementListSyntax {
+  public init(expressions: [ExprSyntaxProtocol]) {
```

Comment:
You should explicitly write `any ExprSyntaxProtocol` here.

Reply:
We prefer to use `ExprSyntax` here. I’m adding some documentation about it in https://github.com/apple/swift-syntax/pull/1696

Reply:
Or that.